### DISCUSSION TITLE
v1.0 komme mit mancher Bedienung nicht klar

### DISCUSSION THREAD

**Discussion by cmjub** [Stellarium/stellarium#2727] [Q&A] [answered]
Hallo Zusammen,

ich habe mir natürlich gleich die neue v1.0 installiert.
Was ich damit irgendwie nicht mehr schaffe ist, das z.B. Ost senkrecht nach unten weißt (so, als wenn ich nach Osten schaue).
Es bleibt immer so, als wenn ich nach Süden schaue, die anderen Bereiche werden dann entsprechend verzerrt.

In den Einstellungen habe ich bis jetzt nichts gefunden, wo/wie ich das ändern kann

Wenn Da jemand helfen kann???

Vielen Dank schon mal
Jens-Uwe

**Comment by 10110111** [comment, marked as answer]
You can drag the image with the mouse, and view direction will change accordingly.

**Reply by cmjub** [reply]
Genau DAS klappt komischerweise bei der v1.0 (bei MIR) nicht mehr🤔

**Comment by gzotti** [comment]
F1 bietet eine Liste der Tastendrücke. Shift+E (engl.: "East"=dt. "Ost") ist die Voreinstellung für "Blick nach Osten". 
Sollte der Zenit im Zentrum stehen sollen, und von da eine vertikale Linie nach unten zum Ostpunkt, dann eben zuerst Shift+Z, dann Shift+E drücken. Das war auch schon früher so.

**Reply by cmjub** [reply]
![image](https://user-images.githubusercontent.com/114985472/194336570-01da7666-0036-4e9b-804f-d182751ec5bd.png)
dann mache ich Shift+E und das Bild wird nur nach rechts verschoben, aber nicht perspektivisch angepaßt
![image](https://user-images.githubusercontent.com/114985472/194337004-e7817837-1486-4535-b269-a5cede3dfa70.png)

ich hoffe, die Bilder werden angezeigt.

Habe ich vielleicht eine falsche Perspektive eingestellt??? Aber mit dem "alten" Stellarium hat es geklappt...

Vielen Dank aber für die Hilfe
Jens-Uwe

**Reply by gzotti** [reply]
Ich weiß ja nicht wie alt ihr "altes" war. Vielleicht benennen Sie einmal die config.ini um, dann wird eine neue Version erzeugt. Der Speicherort wird im User Guide, Kapitel 5, beschrieben.

**Reply by alex-w** [reply]
`Ctrl+M` ?
